### DISCUSSION TITLE
Multiple command triggers in one button

### DISCUSSION THREAD

**Discussion by 4164696f73** [Chatterino/chatterino2#4319] [Ideas]
We can set our own command buttons by the users name, similar to default Twitch chat (ban, timeout 10 min, remove message) (username), but I can't seem to find a way to add "do x or y" command, for example "ban or unban if already banned", same as in regular Twitch chat where you can press the ban button twice and you first ban the user, then unban. 

Something like that would be helpful in case you missclick on someone and accidentally ban them, alternative way is to add both buttons by the name, but that takes too much space. You can press on the user's name and then unban, or type the command, but that is slower to do.
This could be implemented similarly to filters, where you set a condition, if user is not banned/timed out, you ban/time them out, if user is banned/timed out, you do something else (e.g. unban, send message).

Alternatively, allow us to position the buttons how we want them to have, so for example buttons over the name instead of by the name. Visualisation:
`unban ban 10m 1s`
`[timestamp] user.name: user.msg`
instead of :
`[timestamp] unban ban 10m 1s user.name: user.msg`
This way you could have any number of timeout methods and still have nicely readable chat.

**Comment by pajlada** [comment]
Conditional buttons would be complicated to get right as we might not have immediate access to their current state (i.e. if they're banned, timed out, or neither) so we wouldn't be able to accurately show what the button would do.

I would be ok with a setting to render your moderation buttons on a line above the message instead.
If this is an acceptable compromise for you, please go ahead and [create an Issue](https://github.com/Chatterino/chatterino2/issues/new?assignees=&labels=issue-report) with that specific feature request
